SPECS/coreutils/coreutils.spec

```diff
@@ -92,7 +87,6 @@ LANGUAGE=en_US.UTF-8 LC_ALL=en_US.UTF-8 make -k check
 %files
 %defattr(-,root,root)
 %license COPYING
-/bin/*
```

Comment:
When I think about it now: what's the problem with `/bin`? It seems we have now 25 packages providing something under the `/bin` path, 33 requiring something from that path for run-time and 2 for build-time. Some from the same spec (18 core specs in total). Some of the specs we import from other distros might also have similar dependencies.

With that in mind, I'm curious, if it wouldn't be safer for the duration of 2.0 to provide both `/bin` and `/usr/bin` instead of updating the dependencies. Would it work, if you linked `ln -s /usr/bin %{buildroot}/bin` and left the `/bin/*` in the `%files` section intact? It's possible you might need to also `%exclude /bin` in case the build will complain that the directory itself is not listed.

Reply:
I agree is a risky change and I am going to close this PR and create a new one with changes you suggested